multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.110.0-persistence.xml

```diff
@@ -15,9 +15,7 @@
                 <constraints primaryKey="true" primaryKeyName="file_id_pk" nullable="false" />
             </column>
             <column name="content" type="${large-blob.type}" />
-            <column name="digest" type="VARCHAR(255)">
-                <constraints nullable="false" />
-            </column>
+            <column name="digest" type="VARCHAR(255)" />
```

Comment:
You can't do this, as it will break the migration on existing databases. Modifying an existing Liquibase changelog is very dangerous - see https://www.liquibase.org/blog/dealing-with-changing-changesets. You need to create a new changelog - `db-changelog-1.115.0-persistence.xml` and use the following change there:
https://docs.liquibase.com/change-types/community/drop-not-null-constraint.html